deploy/mainnet/460_deploy_SUSDMetaPoolV3.ts

```diff
@@ -0,0 +1,92 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+import { MULTISIG_ADDRESSES } from "../../utils/accounts"
+
+// Deployment names
+const META_POOL_NAME = "SaddleSUSDMetaPoolV3"
+const META_POOL_LP_TOKEN_NAME = `${META_POOL_NAME}LPToken`
+const BASE_POOL_NAME = "SaddleUSDPoolV2"
+
+// Constructor arguments
+const TOKEN_NAMES = ["SUSD", `${BASE_POOL_NAME}LPToken`]
+const TOKEN_DECIMALS = [18, 18]
+const LP_TOKEN_NAME = "Saddle sUSD/saddleUSD-V2"
```

Comment:
I believe this should be V3 correct? this same LP token name was deploy with the SUSD 'Updated' Metapool